src/Controller/ItemsController.ts

```diff
@@ -28,11 +34,25 @@ export class ItemsController extends BaseHttpController {
       apiVersion: request.body.api,
     })
 
-    await this.postToRealtimeExtensions.execute({
-      userUuid: response.locals.user.uuid,
-      itemHashes: request.body.items,
-    })
+    try {
+      await this.postToRealtimeExtensions.execute({
```

Comment:
Will this await the full execution of the job, or just queuing the job?

Reply:
If awaiting full execution that would be unideal as communicating with 3rd party clouds could take arbitrarily long.

Reply:
Adding event to the queue. In essence this replicates: https://github.com/standardnotes/syncing-server/blob/develop/app/controllers/api/items_controller.rb#L17-L20

Reply:
> If awaiting full execution that would be unideal as communicating with 3rd party clouds could take arbitrarily long.

I would know ;)

Reply:
Ah cool 👌